core/scss/components/date-picker/module.scss

```diff
@@ -1,4 +1,7 @@
 @import '../../tools/functions/core';
+@import '../../tools/mixins/core';
+
+@import '../../../../icons/scss/mixins.scss';
```

Comment:
Use this `@import '~@momentum-ui/icons/scss/mixins';` instead of the relative path.

Reply:
got it